examples/base/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@raincatcher/base",
+  "version": "1.0.0",
+  "description": "NPM package template for the RainCatcher project",
+  "main": "index.js",
+  "author": "feedhenry-raincatcher@redhat.com",
+  "license": "Apache-2.0",
+  "scripts": {
+    "build": "del lib && tsc",
+    "test": "nyc mocha",
+    "prepublish": "npm run build"
+  },
+  "nyc": {
+    "include": [
+      "src/**/*.ts"
+    ],
+    "extension": [
+      ".ts"
+    ],
+    "require": [
+      "ts-node/register"
+    ],
+    "check-coverage": true,
+    "lines": 75,
+    "functions": 100,
+    "branches": 80
```

Comment:
@vchepeli @nialldonnellyfh [RAINCATCH-976](https://issues.jboss.org/browse/RAINCATCH-976) doesn't have a definition for functions, we could either go with the global requirement for 80% or go for 100% for functions.

Also I don't know what's the best config for istanbul's reporters, so just leaving the default right now, would love some input.